webview/src/experiments/components/table/Indicators.tsx

```diff
@@ -139,6 +146,7 @@ export const Indicators = () => {
           branchesSelected,
           'Selected'
         )}
+        disabled={!hasBranchesToSelect}
```

Comment:
[Q] Can we give the same branch icon next to the branch name the same action? We could even extend the behaviour on to the branch name. I.e if the user clicks on the branch icon or the branch name then the picker is launched. Having the + - closer may make it more obvious that all of this is clickable.

Reply:
I don't know. We're moving all actions to icons here; making the branch name clickable at the same time might be counterintuitive. The goal of this PR was also to remove the generic action from repeating on all branches. Maybe it'll work still.  I'll explore this in a follow-up.